tests/specs/create-paths-matcher.ts

```diff
@@ -31,15 +31,25 @@ export default testSuite(({ describe }) => {
 					'tsconfig.json': createTsconfigJson({
 						compilerOptions: {
 							paths: {
-								'@': ['src'],
+								'@/*': ['./src/*'],
```

Comment:
Please revert the configuration change. We want to handle the case where users pass in a path without `./`.

Reply:
To be clear, `baseUrl` can only be absent if `paths` is relative.

Reply:
You can add another test for that. Changing this one reduces coverage.